spec/orbital/remote/integration_spec.rb

```diff
@@ -21,13 +22,14 @@
 
   context 'credit card flow' do
     before(:each) do
-      @properties = build_pm_properties(nil, { :cc_number => '5454545454545454' })
+      @properties = build_pm_properties(nil, { :cc_number => '4444444444444448' })
```

Comment:
Are you changing all tests on purpose to use a Visa card? Or should this only be done for these new MIT CIT ones?

Reply:
Not all the tests are changed to VISA because we still have tests that are card type dependent (see those integration tests for network tokenized cards). For other tests, yes, the default test cards are now changed to VISA; I guess it doesn't matter much because for these tests the expected behavior doesn't depend on the card type.

Reply:
Sounds good 👍